### PR TITLE
fix(DB/ElwynnForest): remove SAI from Defias Cutpurse

### PR DESCRIPTION
**Pull request by elthehablo**
Related:
- https://github.com/azerothcore/azerothcore-wotlk/pull/16471
- https://github.com/azerothcore/azerothcore-wotlk/issues/16433

## Changes Proposed:
-  remove broken SAI from defias cutpurse
-  

## Issues Addressed:

- Closes 

## SOURCE:

See discussion on PR. tl;dr: backstab ability existed in original wotlk (very likely) got removed with class as a nerf to the starting zones to make them more noob friendly

## Tests Performed:

- not tested but db checked
- 

## How to Test the Changes:

1. check defias cutpurse
2. should have no differences to before because the backstab was not working
3.

## Known Issues and TODO List:

none

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
> tl;dr: backstab ability existed in original wotlk (very likely) got removed with class as a nerf to the starting zones to make them more noob friendly

That is not my belief to be clear, I believe that the spell was removed at some point with original Wrath, not as a Classic change.

**Comment by Gultask** [comment]
I believe that they have the spell but can't cast because they have no energy and this spells requires it

**Comment by Gultask** [comment]
image

**Comment by heyitsbench** [comment]
> I believe that they have the spell but can't cast because they have no energy and this spells requires it

Wouldn't the spell either not require energy or the cutpurse have energy on Classic Era for them to be able to cast it then? Because neither is true.

**Comment by Gultask** [comment]
image

ok

**Review by Gultask — APPROVED** [review]
